truss/tests/templates/control/control/helpers/test_requirement_name_identifier.py

```diff
@@ -29,6 +29,7 @@ def test_identify_requirement_name(req, expected_name):
 def test_reqs_by_name():
     reqs = [
         "pytorch",
+
```

Comment:
you probably wanna add an empty string here if you wanna test this.

Reply:
oops that's probably ideal